src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/_help.py

```diff
@@ -10,4 +10,12 @@
 helps['configure'] = """
             type: command
             short-summary: Configure Azure CLI 2.0 Preview or view your configuration. The command is interactive, so just type `az configure` and respond to the prompts.
+            examples:
+                - name: configure a default resource group
+                  text: >
+                    az configure --setting resource_group myRG
+                - name: configure a default webapp
+                  text: >
+                    az configure --section compute/vm --value myvm
```

Comment:
For the same reason I mentioned for group and appservice/web, I'd love to discuss whether this could just be "vm" instead of "compute/vm", since the former would match the "resource hierarchy" of the respective noun in the CLI.

From a discoverability perspective, having the setting names match their corresponding command group structure seems like it could help make them more predictable.